executor/common.h

```diff
@@ -118,7 +159,9 @@ __attribute__((noreturn)) void exitf(const char* msg, ...)
 	fprintf(stderr, " (errno %d)\n", e);
 	doexit(kRetryStatus);
 }
+#endif
 
+#if defined(SYZ_EXECUTOR) || defined(SYZ_DEBUG)
```

Comment:
Surrounding all debug calls with `#if defined(SYZ_EXECUTOR) || defined(SYZ_DEBUG)/#endif` produces lots of clutter. Please try to find a way to not require that.